qa/standalone/ceph-helpers.sh

```diff
@@ -582,7 +582,7 @@ function run_osd() {
 
     mkdir -p $osd_data
     ceph-disk $ceph_disk_args \
-        prepare --filestore $osd_data || return 1
+        prepare --filestore $osd_data --verbose || return 1
```

Comment:
this will not work for `prepare` because `--verbose` is a global (top level) argument. So what you need here is:

    ceph-disk --verbose $ceph_disk_args \
        prepare --filestore $osd_data || return 1